directed_forgetting/experiment.js

```diff
@@ -66,6 +66,48 @@ var getInstructFeedback = function() {
 		'</p></div>'
 }
 
+var getTestFeedback = function() {
+	var global_trial = jsPsych.progress().current_trial_global
+	var data_length = 144
+	var start_cut = global_trial - data_length
+	
+	//below are counters to see if the subject is treating this task as a directed remembering as opposed to a directed forgetting task
+	var neg_respond_remember_total = 0
+	var pos_respond_remember_total = 0
+	var neg_respond_remember = 0
+	var pos_respond_remember = 0
+	
+	for(var b =0; b < data_length; b++){
+		if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).probe_type == 'neg'){
+		neg_respond_remember_total += 1
+			if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).key_press == '37'){
+			neg_respond_remember += 1
```

Comment:
So this would just be "remember_total += 1" If you make the change from my last commit